### PR TITLE
docs: discussions disabled

### PR DESCRIPTION
**Pull request by jolheiser**
As title

/cc @sgoudham @nekowinston

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,7 +126,5 @@ See [CONTRIBUTING.md](docs/CONTRIBUTING.md)
 </details>
 <!-- AUTOGEN:USERSTYLES END -->
```
Comment (jolheiser) [inline README.md:128]:
```suggestion
&nbsp;

```

File: README.md [hunk 2]
```diff
@@ -125,7 +125,6 @@ See [CONTRIBUTING.md](docs/CONTRIBUTING.md)
 
 </details>
 <!-- AUTOGEN:USERSTYLES END -->
```
Comment (jolheiser) [inline README.md:127]:
```suggestion
<!-- AUTOGEN:USERSTYLES END -->

```

**Review by nekowinston — APPROVED** [review]

**Review by nekowinston — APPROVED** [review]
Perfect for gorgeous looks, should merge asap @sgoudham

**Review by sgoudham — CHANGES_REQUESTED** [review]
LGTM!

**Review by sgoudham — APPROVED** [review]

**Comment by sgoudham** [comment]
Verifying the pull request and merging now @nekowinston

**Pull request merged**